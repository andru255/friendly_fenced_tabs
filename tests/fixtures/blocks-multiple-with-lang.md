```python
print "Hi Im' from python 2.6"
```

```python
print ("Hi Im' from python 3.6")
```
