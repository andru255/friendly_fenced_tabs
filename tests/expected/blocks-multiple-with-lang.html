<div class="friendly_fenced_tabs">
    <ul class="friendly_fenced_tabs__header">
        <li class="friendly_fenced_tabs__title">
            Python
        </li>
        <li class="friendly_fenced_tabs__title">
            Python
        </li>
    </ul>
    <ul class="friendly_fenced_tabs__content">
        <li class="friendly_fenced_tabs__item">
            <pre><code>print "Hi Im' from python 2.6"</code></pre>
        </li>
        <li class="friendly_fenced_tabs__item">
            <pre><code>print ("Hi Im' from python 3.6")</code></pre>
        </li>
    </ul>
</div>